services/lock-keeper/lockkeeper.py

```diff
@@ -104,3 +106,21 @@ def ports_up():
 def ports_down():
     change_ports_state(request.get_json(), "down")
     return jsonify({'status': 'ok'})
+
+
+@app.route('/floodlight/stop', methods=['GET'])
```

Comment:
Maybe it should be POST instead of GET?

Reply:
What? why?

Reply:
Because GET is intended to take some information about but not doing any actions. I am not saying that it should be exactly POST but not GET for sure.
methods=['GET'] -> methods=['POST']